### ISSUE TITLE
[Quest] Timber Worg Tails

### ISSUE THREAD

**Issue by davidmurcia** [FireFly-WoW/FireFly-IssueTracker#259]
**Description:**

Timber Wold Tails for this quest:
http://www.wowhead.com/quest=10016/timber-worg-tails

Have a wrong drop rate: 
http://www.wowhead.com/item=25807/timber-worg-tail
They should have 23% on the mobss I was killing. 
**Current behaviour:**

```
[13:40] Timber Worg Alpha dies, you gain 578 experience.
[13:40] You receive loot: [Bloody Fang].
[13:40] You receive loot: [Knothide Leather].
[13:40] Timber Worg Alpha dies, you gain 578 experience.
[13:40] You receive loot: [Severed Claw].
[13:40] You receive loot: [Knothide Leather Scraps].
[13:40] Timber Worg Alpha dies, you gain 578 experience.
[13:40] You receive loot: [Bloody Fang].
[13:40] You receive loot: [Severed Claw].
[13:40] Your skill in Skinning has increased to 371.
[13:40] You receive loot: [Knothide Leather].
[13:41] Timber Worg Alpha dies, you gain 550 experience.
[13:41] You receive loot: [Bloody Fang].
[13:41] You receive loot: [Knothide Leather Scraps].
[13:41] You receive loot: [Knothide Leather Scraps].
[13:47] Your skill in Skinning has increased to 372.
[13:47] You receive loot: [Knothide Leather Scraps].
[13:48] Timber Worg Alpha dies, you gain 578 experience.
[13:48] You receive loot: [Bloody Fang].
[13:48] You receive loot: [Knothide Leather Scraps].
[13:48] Timber Worg Alpha dies, you gain 550 experience.
[13:48] You receive loot: [Bloody Fang].
[13:48] You receive loot: [Knothide Leather].
[13:48] Timber Worg Alpha dies, you gain 550 experience.
[13:48] You receive loot: [Bloody Fang].
[13:48] You receive loot: [Knothide Leather].
[13:50] Timber Worg Alpha dies, you gain 550 experience.
[13:50] You receive loot: [Knothide Leather Scraps].
[13:47] Your skill in Skinning has increased to 372.
[13:47] You receive loot: [Knothide Leather Scraps].
[13:48] Timber Worg Alpha dies, you gain 578 experience.
[13:48] You receive loot: [Bloody Fang].
[13:48] You receive loot: [Knothide Leather Scraps].
[13:48] Timber Worg Alpha dies, you gain 550 experience.
[13:48] You receive loot: [Bloody Fang].
[13:48] You receive loot: [Knothide Leather].
[13:48] Timber Worg Alpha dies, you gain 550 experience.
[13:48] You receive loot: [Bloody Fang].
[13:48] You receive loot: [Knothide Leather].
[13:50] Timber Worg Alpha dies, you gain 550 experience.
[13:50] You receive loot: [Knothide Leather Scraps].
[13:55] Timber Worg Alpha dies, you gain 578 experience.
[13:55] You receive loot: [Knothide Leather Scraps].
[13:55] Your skill in Defense has increased to 313.
[13:56] Timber Worg Alpha dies, you gain 578 experience.
[13:56] You receive loot: [Knothide Leather].
[13:56] Timber Worg Alpha dies, you gain 550 experience.
[13:56] You receive loot: [Bloody Fang].
[13:56] You receive loot: [Timber Worg Tail].
[13:56] Your skill in Skinning has increased to 373.
[13:56] You receive loot: [Knothide Leather Scraps].
```

![image](https://cloud.githubusercontent.com/assets/2714797/25849488/f90d30ce-34be-11e7-9ed8-1c3069317852.png)

edit: Left only Alfa wolve kills on the log.

**Comment by TW1NKLET** [comment]
> I would like to request an output of the following query.
```SQL
SELECT * FROM `creature_loot_template` WHERE `item` = 25807;
```


> However, this is fresh data from the latest revision of **T**rinity**C**ore.
```SQL
UPDATE `creature_loot_template` SET `Chance` = 25 WHERE `Item` = 25807 AND `Entry` IN(18476, 18670);
UPDATE `creature_loot_template` SET `Chance` = 15 WHERE `Item` = 25807 AND `Entry` IN(18477);
```

**Comment by JunkyBulgaria** [comment]
fixed as of #261
